packages/blockchain-link/package.json

```diff
@@ -19,49 +19,37 @@
         "javascript"
     ],
     "main": "./lib/index.js",
-    "files": [
-        "lib/",
-        "build/"
-    ],
+    "files": ["lib/"],
     "types": "lib/index.d.ts",
     "typings": "lib/index.d.ts",
     "browser": {
         "ws": "./lib/utils/ws.js"
     },
     "scripts": {
-        "dev": "webpack serve --config ./webpack/dev.babel.js",
-        "build:lib": "yarn build && yarn build:workers",
-        "build:workers": "rimraf build && yarn build:workers-web && yarn build:workers-node && yarn build:workers-module",
-        "build:workers-web": "webpack --config ./webpack/workers.web.babel.js",
-        "build:workers-node": "webpack --config ./webpack/workers.node.babel.js",
-        "build:workers-module": "webpack --config ./webpack/workers.module.babel.js",
-        "lint": "eslint '**/*{.ts,.tsx}'",
+        "dev": "webpack serve --config ./webpack/dev.js",
+        "dev:module": "USE_MODULES=true webpack serve --config ./webpack/dev.js",
+        "build:lib": "rimraf lib && tsc --p ./tsconfig.lib.json && tsc --p ./tsconfig.workers.json",
+        "build:workers": "rimraf build && yarn build:workers-web && yarn build:workers-module",
+        "build:workers-web": "webpack --config ./webpack/workers.web.js",
+        "build:workers-module": "webpack --config ./webpack/workers.module.js",
+        "lint": "eslint '**/*.ts'",
         "test:unit": "jest --verbose -c jest.config.unit.js",
         "test:integration": "jest -c jest.config.integration.js",
-        "build": "rimraf lib && tsc --p ./tsconfig.lib.json && tsc --p ./tsconfig.workers.json",
         "type-check": "tsc --project tsconfig.json"
     },
     "devDependencies": {
-        "@babel/register": "^7.13.14",
-        "@babel/runtime": "^7.13.10",
-        "babel-eslint": "^10.1.0",
-        "babel-loader": "^8.2.1",
-        "eslint-config-airbnb-base": "^14.2.1",
-        "eslint-loader": "^4.0.2",
-        "eslint-plugin-jest": "^24.4.0",
         "html-webpack-plugin": "^5.3.2",
+        "tiny-worker": "^2.3.0",
         "ts-loader": "^9.2.5",
         "webpack": "^5.55.1",
         "webpack-cli": "4.8.0",
-        "webpack-dev-server": "^4.2.1",
+        "webpack-dev-server": "^4.7.3",
         "worker-loader": "^3.0.8"
     },
     "dependencies": {
         "bignumber.js": "^9.0.1",
-        "es6-promise": "^4.2.8",
-        "events": "^3.2.0",
+        "events": "^3.3.0",
         "ripple-lib": "1.10.0",
-        "tiny-worker": "^2.3.0",
         "ws": "^7.4.0"
```

Comment:
Could you please try to upgrade `ws` to version 7.5.5 or higher? We currently have like 5 different version of `ws` in project and I want to remove at least some of them. If this will cause problems then we can leave it as it is, but 5 deferent versions is quite a lot.

![Screenshot 2022-01-18 at 12 33 53](https://user-images.githubusercontent.com/5837757/149930349-23e9401b-27c5-4c8f-92fa-33e428240712.png)

Also related to #4760

Reply:
i think it should be done in separate PR, `ws` needs to be updated across the whole monorepo packages not only blockchain-link and this PR is about blockchain-link modules :) i already ended up with much more code changes than i wanted

Reply:
thanks for explanation, resolving for now